src/iaux-underlined-tab-bar-loading-dots.ts

```diff
@@ -0,0 +1,75 @@
+import { html, css, LitElement, customElement, CSSResult } from 'lit-element';
+
+@customElement('iaux-underlined-tab-bar-loading-dots')
+export class LoadingDots extends LitElement {
+  render() {
+    return html`
+      <div class="spinnyCircle">
+        <div></div>
```

Comment:
since these dots are laid out inline, would it be more semantic to use `span` instead of `div`? for example see the discussion here: https://stackoverflow.com/questions/183532/what-is-the-difference-between-html-tags-div-and-span